docs(RadioGroup): update focus documentation (#2903)
telerik/blazor-docs: components/radiogroup/overview.md

## RadioGroup Reference and Methods

> The RadioGroup provides a `FocusItemAsync` method that allows the application to focus a specific radio input programmatically. It accepts an integer parameter representing the index of the radio button to focus. To use it is required to obtain the component reference through its `@ref` attribute.

Reviewer: ```suggestion
The RadioGroup provides a `FocusItemAsync` method that allows the application to focus a specific radio input programmatically. It accepts an integer parameter representing the index of the radio button you want to focus. To use it, obtain the component reference through its `@ref` attribute.
```

> >caption Using RadioGroup FocusItemAsync method

Reviewer: ```suggestion
>caption Using RadioGroup `FocusItemAsync` method
```
